users/noroadsleft/noroadsleft.c

```diff
@@ -73,60 +68,35 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
             return false;
         case M_SALL:
             if (record->event.pressed) {
-                if ( macroMode == 1 ) {
-                    SEND_STRING(SS_LGUI("a"));
-                } else {
-                    SEND_STRING(SS_LCTL("a"));
-                }
+                tap_code16(C(KC_A));
```

Comment:
Couldn't this and others be replaced in the keymap?

Reply:
This is deliberate.

https://github.com/qmk/qmk_firmware/blame/7a8fa90c4f6a44043343ff7194928a0040b41ca0/users/noroadsleft/readme.md#L32:

> I used to have a boolean variable that changed the functionality of these keycodes, but I no longer work in the environment that I wrote the functionality for, so I took it out. The keycodes still exist because all my `keymap.c` files reference the custom keycodes I defined.

I found it less involved to just rewrite the keycodes, versus actually replacing them with native `C(KC_A)`, etc. Makes it easier to revert if I need the functionality again.